File: PHP/PerformanceTest.php
<?php
    $start = microtime(true);
    for ($x=0; $x<1000000; $x++) {
      $str = 'aaaaaaaaaa';
      $hash = hash('sha256', $str);
      //echo($hash);
      //echo("<br/>");
    }
    $end = microtime(true);
    echo("<br/>");
    echo("<br/>");
    echo("<br/>");
    echo(($end - $start)*1000000);
?>

File: C#/PerformanceTest.cs
using System;
using System.Diagnostics;
using System.Security.Cryptography;
using System.Text;

namespace PerformanceTest
{
    class PerformanceTest
    {
        static void Main(string[] args)
        {
            string data = "aaaaaaaaaa";
            Stopwatch time = new Stopwatch();
            time.Start();

            for (int i = 0; i < 1000000; i++)
            {
                try
                {
                    string hashvalue = ComputeSha256Hash(data);
                }
                catch (Exception)
                {
                    throw;
                }
            }
            time.Stop();
            Debug.WriteLine("Duration: " + (long)time.Elapsed.Milliseconds * 1000);
        }

        private static string ComputeSha256Hash(string rawData)
        {
            using (SHA256 sha256Hash = SHA256.Create())
            {
                byte[] bytes = sha256Hash.ComputeHash(Encoding.UTF8.GetBytes(rawData));

                StringBuilder builder = new StringBuilder();
                for (int i = 0; i < bytes.Length; i++)
                {
                    builder.Append(bytes[i].ToString("x2"));
                }
                return builder.ToString();
            }
        }
    }
}
